haproxy16/plan.sh

```diff
@@ -0,0 +1,12 @@
+if [[ $(caller) = *"test"* ]] || [[ $(caller) = *"bats"* ]]; then
```

Comment:
This works when calling from top level (in core-plans directory) but fails if you are inside the haproxy16 directory during build. The path may need to be relative using `BASH_SOURCE` or similar.